### PR TITLE
Added documentation about mac address and virt-win

### PR DESCRIPTION
**Pull request by SchSeba**
Related to https://github.com/kubevirt/kubevirt/issues/1850#issuecomment-448792407 added some documentation about mac address configuration in the bridge interface type.

@fabiand @justinbarrick can you please take a look

---
This change is [Reviewable](https://reviewable.io/reviews/kubevirt/user-guide/205)

### CODE REVIEW

File: creating-virtual-machines/interfaces-and-networks.adoc [hunk 1]
```diff
@@ -282,6 +282,8 @@ In `bridge` mode, virtual machines are connected to the network backend
 through a linux ``bridge''. The pod network IPv4 address is delegated to
 the virtual machine via DHCPv4. The virtual machine should be configured
 to use DHCP to acquire IPv4 addresses.
+MAC address is also delegated if a specific MAC is not configured in the vm spec
+using DHCPv4 option 26 and also via the virtual machine domxml.
```
Comment (rmohr) [inline creating-virtual-machines/interfaces-and-networks.adoc]: That reads as if we pass through the mac via `26`. Did you write that we also pass through the MTU via dhcp?

**Comment by fabiand** [comment]
@dankenigsberg can you please review?

File: creating-virtual-machines/interfaces-and-networks.adoc (hunk 1 above)
Comment (dankenigsberg) [reply creating-virtual-machines/interfaces-and-networks.adoc]: I share @rmohr puzzlement regarding the English text. It is not clear to me.

@SchSeba can you try to rephrase? Can you also set topic/network label here?

**Comment by SchSeba** [comment]
@dankenigsberg @rmohr Can you please review this again?

File: creating-virtual-machines/interfaces-and-networks.adoc [hunk 2]
```diff
@@ -261,6 +261,12 @@ ______________________________________________________________________________
 drivers.
 ______________________________________________________________________________
 
+______________________________________________________________________________
+*Note:* Windows machines need the latest virtio network driver to configure
+the correct MTU on the interface.
+______________________________________________________________________________
```
Comment (dankenigsberg) [inline creating-virtual-machines/interfaces-and-networks.adoc]: seems unrelated to the purpose of this PR

File: creating-virtual-machines/interfaces-and-networks.adoc [hunk 3]
```diff
@@ -283,6 +289,12 @@ through a linux ``bridge''. The pod network IPv4 address is delegated to
 the virtual machine via DHCPv4. The virtual machine should be configured
 to use DHCP to acquire IPv4 addresses.
 
+________________________________________________________________________________________________________________________________________________________________________________________________________________________________
+**Note:** If a specific MAC address is not configured in the virtual machine interface spec
+the MAC address from the relevant pod interface is delegated to the virtual machine.
+________________________________________________________________________________________________________________________________________________________________________________________________________________________________
```
Comment (dankenigsberg) [inline creating-virtual-machines/interfaces-and-networks.adoc]: text seems fine. I find it hard to see the relevance of the former version of the text, but never mind that. bygone.

File: creating-virtual-machines/interfaces-and-networks.adoc (hunk 2 above)
Comment (SchSeba) [reply creating-virtual-machines/interfaces-and-networks.adoc]: updated the PR name

**Comment by SchSeba** [comment]
@rmohr can you please take a look on this PR
